applications/app/views/fragments/galleryHeader.scala.html

```diff
@@ -2,7 +2,7 @@
 
 @import views.support.TrailCssClasses.toneClass
 
-<header class="content__head tonal__head tonal__head--@toneClass(gallery.item)">
+<header class="content__head tonal__head tonal__head--tone-media tonal__head--@toneClass(gallery.item)">
```

Comment:
Same here. Lets wait for the paid for fapi client change to happen and then we probably won't need to manually add this class.